lib/src/interactions/data/touch_interaction_events.dart

```diff
@@ -1,6 +1,6 @@
-import 'package:chart_it/src/interactions/data/chart_interactions.dart';
+import 'package:chart_it/src/interactions/data/touch_interactions.dart';
 
-abstract class ChartInteractionConfig<T extends ChartInteractionResult> {
+abstract class TouchInteractionEvents<T extends TouchInteractionResult> {
```

Comment:
The reason for not naming this touch is that the chart might not always be on a touch enabled device.